# Problem Index

Problem descriptions and links for this repository's Rosalind solutions.

| ID | Description | Problem | Solution |
| --- | --- | --- | --- |
| `ba1c` | (title unavailable) | [link](https://rosalind.info/problems/ba1c/) | `./ba1c/main.py` |
| `dna` | Counting DNA Nucleotides | [link](https://rosalind.info/problems/dna/) | `./dna/main.py` |
| `rna` | Transcribing DNA into RNA | [link](https://rosalind.info/problems/rna/) | `./rna/main.py` |
| `revc` | Complementing a Strand of DNA | [link](https://rosalind.info/problems/revc/) | `./revc/main.py` |
| `fib` | Rabbits and Recurrence Relations | [link](https://rosalind.info/problems/fib/) | `./fib/main.py` |
| `gc` | Computing GC Content | [link](https://rosalind.info/problems/gc/) | `./gc/main.py` |
| `hamm` | Counting Point Mutations | [link](https://rosalind.info/problems/hamm/) | `./hamm/main.py` |
| `iprb` | Mendel's First Law | [link](https://rosalind.info/problems/iprb/) | `./iprb/main.py` |
| `prot` | Translating RNA into Protein | [link](https://rosalind.info/problems/prot/) | `./prot/main.py` |
| `subs` | Finding a Motif in DNA | [link](https://rosalind.info/problems/subs/) | `./subs/main.py` |
| `cons` | Consensus and Profile | [link](https://rosalind.info/problems/cons/) | `./cons/main.py` |
| `fibd` | Mortal Fibonacci Rabbits | [link](https://rosalind.info/problems/fibd/) | `./fibd/main.py` |
| `grph` | Overlap Graphs | [link](https://rosalind.info/problems/grph/) | `./grph/main.py` |
| `iev` | Calculating Expected Offspring | [link](https://rosalind.info/problems/iev/) | `./iev/main.py` |
| `lcsm` | Finding a Shared Motif | [link](https://rosalind.info/problems/lcsm/) | `./lcsm/main.py` |
| `lia` | Independent Alleles | [link](https://rosalind.info/problems/lia/) | `./lia/main.py` |
| `mprt` | Finding a Protein Motif | [link](https://rosalind.info/problems/mprt/) | `./mprt/main.py` |
| `mrna` | Inferring mRNA from Protein | [link](https://rosalind.info/problems/mrna/) | `./mrna/main.py` |
| `orf` | Open Reading Frames | [link](https://rosalind.info/problems/orf/) | `./orf/main.py` |
| `perm` | Enumerating Gene Orders | [link](https://rosalind.info/problems/perm/) | `./perm/main.py` |
| `prtm` | Calculating Protein Mass | [link](https://rosalind.info/problems/prtm/) | `./prtm/main.py` |
| `revp` | Locating Restriction Sites | [link](https://rosalind.info/problems/revp/) | `./revp/main.py` |
| `splc` | RNA Splicing | [link](https://rosalind.info/problems/splc/) | `./splc/main.py` |
| `lexf` | Enumerating k-mers Lexicographically | [link](https://rosalind.info/problems/lexf/) | `./lexf/main.py` |
| `lgis` | Longest Increasing Subsequence | [link](https://rosalind.info/problems/lgis/) | `./lgis/main.py` |
| `long` | Genome Assembly as Shortest Superstring | [link](https://rosalind.info/problems/long/) | `./long/main.py` |
| `pmch` | Perfect Matchings and RNA Secondary Structures | [link](https://rosalind.info/problems/pmch/) | `./pmch/main.py` |
| `pper` | Partial Permutations | [link](https://rosalind.info/problems/pper/) | `./pper/main.py` |
| `prob` | Introduction to Random Strings | [link](https://rosalind.info/problems/prob/) | `./prob/main.py` |
| `sign` | Enumerating Oriented Gene Orderings | [link](https://rosalind.info/problems/sign/) | `./sign/main.py` |
| `sseq` | Finding a Spliced Motif | [link](https://rosalind.info/problems/sseq/) | `./sseq/main.py` |
| `tran` | Transitions and Transversions | [link](https://rosalind.info/problems/tran/) | `./tran/main.py` |
| `tree` | Completing a Tree | [link](https://rosalind.info/problems/tree/) | `./tree/main.py` |
| `cat` | Catalan Numbers and RNA Secondary Structures | [link](https://rosalind.info/problems/cat/) | `./cat/main.py` |
| `corr` | Error Correction in Reads | [link](https://rosalind.info/problems/corr/) | `./corr/main.py` |
| `inod` | Counting Phylogenetic Ancestors | [link](https://rosalind.info/problems/inod/) | `./inod/main.py` |
| `kmer` | k-Mer Composition | [link](https://rosalind.info/problems/kmer/) | `./kmer/main.py` |
| `kmp` | Speeding Up Motif Finding | [link](https://rosalind.info/problems/kmp/) | `./kmp/main.py` |
| `lcsq` | Finding a Shared Spliced Motif | [link](https://rosalind.info/problems/lcsq/) | `./lcsq/main.py` |
| `lexv` | Ordering Strings of Varying Length Lexicographically | [link](https://rosalind.info/problems/lexv/) | `./lexv/main.py` |
| `mmch` | Maximum Matchings and RNA Secondary Structures | [link](https://rosalind.info/problems/mmch/) | `./mmch/main.py` |
| `pdst` | Creating a Distance Matrix | [link](https://rosalind.info/problems/pdst/) | `./pdst/main.py` |
| `rear` | Reversal Distance | [link](https://rosalind.info/problems/rear/) | `./rear/main.py` |
| `rstr` | Matching Random Motifs | [link](https://rosalind.info/problems/rstr/) | `./rstr/main.py` |
| `sset` | Counting Subsets | [link](https://rosalind.info/problems/sset/) | `./sset/main.py` |
| `aspc` | Introduction to Alternative Splicing | [link](https://rosalind.info/problems/aspc/) | `./aspc/main.py` |
| `edit` | Edit Distance | [link](https://rosalind.info/problems/edit/) | `./edit/main.py` |
| `eval` | Expected Number of Restriction Sites | [link](https://rosalind.info/problems/eval/) | `./eval/main.py` |
| `motz` | Motzkin Numbers and RNA Secondary Structures | [link](https://rosalind.info/problems/motz/) | `./motz/main.py` |
| `nwck` | Distances in Trees | [link](https://rosalind.info/problems/nwck/) | `./nwck/main.py` |
| `scsp` | Interleaving Two Motifs | [link](https://rosalind.info/problems/scsp/) | `./scsp/main.py` |
| `seto` | Introduction to Set Operations | [link](https://rosalind.info/problems/seto/) | `./seto/main.py` |
| `sort` | Sorting by Reversals | [link](https://rosalind.info/problems/sort/) | `./sort/main.py` |
| `spec` | Inferring Protein from Spectrum | [link](https://rosalind.info/problems/spec/) | `./spec/main.py` |
| `trie` | Introduction to Pattern Matching | [link](https://rosalind.info/problems/trie/) | `./trie/main.py` |
| `conv` | Comparing Spectra with the Spectral Convolution | [link](https://rosalind.info/problems/conv/) | `./conv/main.py` |
| `ctbl` | Creating a Character Table | [link](https://rosalind.info/problems/ctbl/) | `./ctbl/main.py` |
| `dbru` | Constructing a De Bruijn Graph | [link](https://rosalind.info/problems/dbru/) | `./dbru/main.py` |
| `edta` | Edit Distance Alignment | [link](https://rosalind.info/problems/edta/) | `./edta/main.py` |
| `full` | Inferring Peptide from Full Spectrum | [link](https://rosalind.info/problems/full/) | `./full/main.py` |
| `indc` | Independent Segregation of Chromosomes | [link](https://rosalind.info/problems/indc/) | `./indc/main.py` |
| `itwv` | Finding Disjoint Motifs in a Gene | [link](https://rosalind.info/problems/itwv/) | `./itwv/main.py` |
| `lrep` | Finding the Longest Multiple Repeat | [link](https://rosalind.info/problems/lrep/) | `./lrep/main.py` |
| `nkew` | Newick Format with Edge Weights | [link](https://rosalind.info/problems/nkew/) | `./nkew/main.py` |
| `rnas` | Wobble Bonding and RNA Secondary Structures | [link](https://rosalind.info/problems/rnas/) | `./rnas/main.py` |
| `afrq` | Counting Disease Carriers | [link](https://rosalind.info/problems/afrq/) | `./afrq/main.py` |
| `cstr` | Creating a Character Table from Genetic Strings | [link](https://rosalind.info/problems/cstr/) | `./cstr/main.py` |
| `ctea` | Counting Optimal Alignments | [link](https://rosalind.info/problems/ctea/) | `./ctea/main.py` |
| `cunr` | Counting Unrooted Binary Trees | [link](https://rosalind.info/problems/cunr/) | `./cunr/main.py` |
| `glob` | Global Alignment with Scoring Matrix | [link](https://rosalind.info/problems/glob/) | `./glob/main.py` |
| `pcov` | Genome Assembly with Perfect Coverage | [link](https://rosalind.info/problems/pcov/) | `./pcov/main.py` |
| `prsm` | Matching a Spectrum to a Protein | [link](https://rosalind.info/problems/prsm/) | `./prsm/main.py` |
| `qrt` | Quartets | [link](https://rosalind.info/problems/qrt/) | `./qrt/main.py` |
| `sgra` | Using the Spectrum Graph to Infer Peptides | [link](https://rosalind.info/problems/sgra/) | `./sgra/main.py` |
| `suff` | Encoding Suffix Trees | [link](https://rosalind.info/problems/suff/) | `./suff/main.py` |
| `chbp` | Character-Based Phylogeny | [link](https://rosalind.info/problems/chbp/) | `./chbp/main.py` |
| `cntq` | Counting Quartets | [link](https://rosalind.info/problems/cntq/) | `./cntq/main.py` |
| `eubt` | Enumerating Unrooted Binary Trees | [link](https://rosalind.info/problems/eubt/) | `./eubt/main.py` |
| `gasm` | Genome Assembly Using Reads | [link](https://rosalind.info/problems/gasm/) | `./gasm/main.py` |
| `gcon` | Global Alignment with Constant Gap Penalty | [link](https://rosalind.info/problems/gcon/) | `./gcon/main.py` |
| `ling` | Linguistic Complexity of a Genome | [link](https://rosalind.info/problems/ling/) | `./ling/main.py` |
| `loca` | Local Alignment with Scoring Matrix | [link](https://rosalind.info/problems/loca/) | `./loca/main.py` |
| `mend` | Inferring Genotype from a Pedigree | [link](https://rosalind.info/problems/mend/) | `./mend/main.py` |
| `mgap` | Maximizing the Gap Symbols of an Optimal Alignment | [link](https://rosalind.info/problems/mgap/) | `./mgap/main.py` |
| `mrep` | Identifying Maximal Repeats | [link](https://rosalind.info/problems/mrep/) | `./mrep/main.py` |
| `mult` | Multiple Alignment | [link](https://rosalind.info/problems/mult/) | `./mult/main.py` |
| `pdpl` | Creating a Restriction Map | [link](https://rosalind.info/problems/pdpl/) | `./pdpl/main.py` |
| `root` | Counting Rooted Binary Trees | [link](https://rosalind.info/problems/root/) | `./root/main.py` |
| `sexl` | Sex-Linked Inheritance | [link](https://rosalind.info/problems/sexl/) | `./sexl/main.py` |
| `sptd` | Phylogeny Comparison with Split Distance | [link](https://rosalind.info/problems/sptd/) | `./sptd/main.py` |
| `wfmd` | The Wright-Fisher Model of Genetic Drift | [link](https://rosalind.info/problems/wfmd/) | `./wfmd/main.py` |
| `alph` | Alignment-Based Phylogeny | [link](https://rosalind.info/problems/alph/) | `./alph/main.py` |
| `asmq` | Assessing Assembly Quality with N50 and N75 | [link](https://rosalind.info/problems/asmq/) | `./asmq/main.py` |
| `cset` | Fixing an Inconsistent Character Set | [link](https://rosalind.info/problems/cset/) | `./cset/main.py` |
| `ebin` | Wright-Fisher's Expected Behavior | [link](https://rosalind.info/problems/ebin/) | `./ebin/main.py` |
| `foun` | The Founder Effect and Genetic Drift | [link](https://rosalind.info/problems/foun/) | `./foun/main.py` |
| `gaff` | Global Alignment with Scoring Matrix and Affine Gap Penalty | [link](https://rosalind.info/problems/gaff/) | `./gaff/main.py` |
| `grep` | Genome Assembly with Perfect Coverage and Repeats | [link](https://rosalind.info/problems/grep/) | `./grep/main.py` |
| `oap` | Overlap Alignment | [link](https://rosalind.info/problems/oap/) | `./oap/main.py` |
| `qrtd` | Quartet Distance | [link](https://rosalind.info/problems/qrtd/) | `./qrtd/main.py` |
| `sims` | Finding a Motif with Modifications | [link](https://rosalind.info/problems/sims/) | `./sims/main.py` |
| `smgb` | Semiglobal Alignment | [link](https://rosalind.info/problems/smgb/) | `./smgb/main.py` |
| `ksim` | Finding All Similar Motifs | [link](https://rosalind.info/problems/ksim/) | `./ksim/main.py` |
| `laff` | Local Alignment with Affine Gap Penalty | [link](https://rosalind.info/problems/laff/) | `./laff/main.py` |
| `osym` | Isolating Symbols in Alignments | [link](https://rosalind.info/problems/osym/) | `./osym/main.py` |
| `rsub` | Identifying Reversing Substitutions | [link](https://rosalind.info/problems/rsub/) | `./rsub/main.py` |
